frontend/app/src/components/display/AmountCurrency.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <fragment>
```

Comment:
Did you get errors about a single root element in this case? I remember having no issues with `v-if`. Still, this is useful until the vue3 update, we have a lot of cases of divs that are used only due to the single root element limitation.

Reply:
Oh I didn't try, I thought that would be an issue, good to know then, do you prefer I change for here?

Reply:
No need, just leave it be, we can use it in other places as well :)